src/styles/components/footer/footer.scss

```diff
@@ -61,6 +76,13 @@ footer {
   font-size: 12px;
 }
 
+.link-group-heading {
+  font-family: $font-family-condensedbold;
+  font-size: 18px;
+  color: $text-color-inverse;
+  text-transform: uppercase;
+}
+
```

Comment:
the whole footer.scss file has to be cleaned up, e.g. link-group-title and link-group-icon are not used any more - let's talk about it

Reply:
css-file has been cleaned up & unused classes have been removed